pom.xml

```diff
@@ -25,9 +25,59 @@
 		<jacoco.out.path>${project.build.directory}</jacoco.out.path>
 		<sonar.jacoco.reportPaths>${jacoco.out.path}/jacoco.exec</sonar.jacoco.reportPaths>
 		<jacoco.version>0.8.2</jacoco.version>
-		<log4j2.version>2.16.0</log4j2.version>
+		<log4j2.version>2.17.2</log4j2.version>
+		<spring.version>5.3.18</spring.version>
```

Comment:
spring version is `5.3.18`, not `5.2.20.RELEASE`?

Reply:
This one has as a base spring-boot 2.4.3, which has spring packages at 5.3.4 version. So the least minor version with the fix is 5.3.18.